import_flow_events.py

```diff
@@ -44,7 +44,7 @@
       timestamp BIGINT NOT NULL SORTKEY,
       type VARCHAR(30) NOT NULL,
       flow_id VARCHAR(64) NOT NULL DISTKEY,
-      flow_time BIGINT NOT NULL,
+      flow_time INTEGER NOT NULL,
```

Comment:
Ah, so it captures how long the flow took? As you say, if the config is adjusted, it doesn't sound that impossible for a flow to overflow 4 bytes. It overflows after around 49 hours, so if a flow was able to take longer than 2 days, it won't fit.